config/config.go

```diff
@@ -208,25 +289,49 @@ func LoadConfig(conffile string) (*Config, error) {
 	return config, err
 }
 
+func (conf *Config) buildPlugins() error {
```

Comment:
I think this name is ambiguous or misleading.
From this name, people may misunderstand it to build plugin-related configs only from `config.Plugin`. But actually this method doesn't erase already existing `conf.MetricPlugins` and `conf.CheckPlugins`, and should not erase them because it's called on `includeConfigFile`.